src/tpl.c

```diff
@@ -253,7 +253,7 @@ static const struct tpl_type_t tpl_types[] = {
 static int tpl_oops(const char *fmt, ...) {
     va_list ap;
     va_start(ap,fmt);
-    vfprintf(stderr,fmt,ap);
+    vprintf(fmt,ap);
```

Comment:
This rather should go to a separate pull request.

Reply:
You are right. I didn't want this change to be part of the pull request for the memory leak issue.

Probably my mistake was creating the pull request on my master branch (I don't work with pull requests). I stopped supporting this pull request because I am not sure this is the correct solution for all cases, as explained above. But for my use case I need the change.

I also need to avoid stderr for my case. I want tpl.lib to be usable in VS versions both before and after 2015. But I do not want this change to make it into the "official" version.

If you don't mind, I would close this pull request.

Reply:
Yes, having static libs compiled with VS < 2015 and linking later with VS 2015 is pain. I also needed to deal with it, see https://github.com/tbeu/ExternData/blob/master/ExternData/Resources/Include/msvc_compatibility.h, but relied on not using stderr at all.